packages/adapters/bitcoin/package.json

```diff
@@ -21,6 +21,26 @@
       "types": "./dist/types/exports/index.d.ts",
       "import": "./dist/esm/exports/index.js",
       "default": "./dist/esm/exports/index.js"
+    },
+    "./react": {
```

Comment:
Revert these as we don't need to return react/vue related things from Bitcoin adapte r